tests/test_astrometryTask.py

```diff
@@ -162,6 +162,17 @@ def doTest(self, pixelsToTanPixels, order=3):
         )
         self.assertLess(len(resultsRefSelect.matches), len(results.matches))
 
+        # try again, allowing magnitude outlier rejection.
+        config.doMagnitudeOutlierRejection = True
+        solverMagOutlierRejection = AstrometryTask(config=config, refObjLoader=self.refObjLoader)
+        self.exposure.setWcs(distortedWcs)
+        resultsMagOutlierRejection = solverMagOutlierRejection.run(
+            sourceCat=sourceCat,
+            exposure=self.exposure,
+        )
+        self.assertLess(len(resultsMagOutlierRejection.matches), len(resultsRefSelect.matches))
```

Comment:
Don't you know exactly how many sources should be rejected, since you explicitly constructed the list? If you're not going to test the actual fit result, at least be more strict about testing the rejections.

Reply:
Relatedly: I'd like the rejection test to check something close to the specific value of the nSigma config field, instead of `1000x`. There's some math going on in the new method, and it would be good to know that `nSigma` means what a user thinks it to mean.